src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -410,7 +417,57 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
             }
         }
 
-        internal void InvokeResponse(InvocationResponse invokeResponse)
+        private async Task<object> GetBindingDataAsync(ParameterBinding binding)
+        {
+            if (binding.RpcSharedMemory != null)
```

Comment:
Should we add some monitoring? Either logs or counters?
We should probably track how many memory maps we create, how big they are, and how many we closed.